src/adstage_api_concepts/date_ranges.apib

```diff
@@ -0,0 +1,30 @@
+## Date Ranges
+
+The AdStage endpoints use date range strings to represent time periods and implement a unified date range format throughout the platform.
+Date range strings can be written in two possible styles: either an absolute range such as `"2016-12-25..2016-12-31"`, or a relative range, such as `"last_30_days"` or `"this_month"`.
```

Comment:
There are a couple of places in the docs, including in examples, where we use the underscored versions of named ranges, though the examples below use spaces instead. Either approach is fine, but we should probably choose one convention and stick to it.

Reply:
So it turns out that the `.apib` parser really doesn't like underscores in certain places. But in other places, it's unacceptable to have spaces written out. I'll take a pass through and see if I can unify it, but as an alternative, I might just add a note explaining that the two are interchangeable.

Reply:
- [x] Standardized with spaces (Looks like everywhere we use them, it should be fine to have a space - it's really JSON keys in some cases that I had trouble with earlier but these are values. Avoiding underscores in general is good for parsing - basically any underscore needs to be escaped by backticks, so this makes writing future docs easier.)